ts/components/core/accordion/RawAccordion.tsx

```diff
@@ -0,0 +1,125 @@
+import { View } from "native-base";
+import * as React from "react";
+import { useEffect, useRef, useState } from "react";
+import {
+  AccessibilityProps,
+  Animated,
+  Easing,
+  LayoutAnimation,
+  Platform,
+  StyleSheet,
+  TouchableOpacity,
+  UIManager
+} from "react-native";
+import I18n from "../../../i18n";
+import themeVariables from "../../../theme/variables";
+import customVariables from "../../../theme/variables";
+import IconFont from "../../ui/IconFont";
+import { IOStyles } from "../variables/IOStyles";
+
+// TODO: handle external initial open/closed state
+type Props = {
+  // The header component, an arrow indicating the open/closed state will be added on the right
+  header: React.ReactElement;
+  // The accordion component must accept one children
+  children: React.ReactElement;
```

Comment:
```suggestion
  child: React.ReactElement;
```
or even `body` could be more expressive?